handlers/fb_handlers.php

```diff
@@ -24,18 +24,20 @@
 // and Facebook
 
 // TODO: this only works if the script is installed in root
-define('FACEBOOK_SDK_V4_SRC_DIR', __DIR__ . '/../include/facebook-php-sdk-v4/src/Facebook/');
-require_once(__DIR__ . '/../include/facebook-php-sdk-v4/autoload.php');
+define('FACEBOOK_SDK_V4_SRC_DIR', __DIR__ . '/../include/php-graph-sdk/src/Facebook');
+require_once(__DIR__ . '/../include/php-graph-sdk/src/Facebook/autoload.php');
```

Comment:
It's not necessary @teldosas. Forgive my lack of attention. Your commit https://github.com/mhaas/fbwlan/pull/22/commits/0e8707c6a91dec631df08bbaf336c8dc0b44a3ea#diff-3aea38d1b59373a8491d3f43bc915f5d deal with this.